v2/protegrity-data-tokenization/src/test/java/com/google/cloud/teleport/v2/templates/ProtegrityDataTokenizationTest.java

```diff
@@ -19,7 +19,6 @@
 import static org.hamcrest.Matchers.hasSize;
```

Comment:
I guess, this file shouldn't be in this PR

Reply:
the line 174 should needs to be changed